src/functions.js

```diff
@@ -1187,6 +1251,10 @@ ${main_content}
             name = game.functions.capitalizeAll(name); // Capitalize all words
 
             let card = game.functions.getCardByName(name + " Starting Hero");
+            if (!card) {
```

Comment:
Also do this if the card isn't a valid starting hero. (Not a hero, doesn't have a `heropower` kwm, etc...)